routing.rst

```diff
@@ -4,38 +4,29 @@
 Routing
 =======
 
-Beautiful URLs are a must for any serious web application. This means leaving
-behind ugly URLs like ``index.php?article_id=57`` in favor of something like
-``/read/intro-to-symfony``.
-
-Having flexibility is even more important. What if you need to change the
-URL of a page from ``/blog`` to ``/news``? How many links would you need to
-hunt down and update to make the change? If you're using Symfony's router,
-the change should be trivial.
-
-.. index::
-   single: Routing; Basics
-
-.. _routing-creating-routes:
+When your application receives an HTTP request, it executes some
+:doc:`controller action </controller>` to generate the HTTP response. The
```

Comment:
Could we remove HTTP in both places? It makes the article sound more technical... not sure it's needed.